tools/docs/config.ts

```diff
@@ -1,25 +1,45 @@
+import stringify from 'json-stringify-pretty-compact';
 import { getOptions } from '../../lib/config/options';
 import { getCliName } from '../../lib/workers/global/config/parse/cli';
 import { getEnvName } from '../../lib/workers/global/config/parse/env';
 import { readFile, updateFile } from '../utils';
 
 const options = getOptions();
 
+function indent(amount: number, indent = '  '): string {
+  let indentation = '';
+  for (let i = 0; i < amount; i++) {
+    indentation += indent;
+  }
+  return indentation;
```

Comment:
Can't we use [string.repeat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat)?

Reply:
That's clean, Didn't know this one. thanks